src/scratchtocatrobat/converter/converter.py

```diff
@@ -579,9 +579,8 @@ def _key_filename_for(key):
     # TODO: extract method, already used once
     return common.md5_hash(key_path) + "_" + _key_to_broadcast_message(key) + os.path.splitext(key_path)[1]
 
-def _generate_mouse_filename():
-    mouse_path = _mouse_image_path()
-    return common.md5_hash(mouse_path) + "_" + MOUSE_SPRITE_FILENAME
+def _get_mouse_filename():
```

Comment:
The call to this function in converter.py line 688 was not adapted to the new name so there is an error when converting a project with mouse cursor. This should be fixed.